spotlight-client/src/Tooltip/Tooltip.tsx

```diff
@@ -18,18 +18,15 @@
 import { rem } from "polished";
 import React from "react";
 import styled from "styled-components/macro";
-import { colors, zIndex } from "../UiLibrary";
+import { Tooltip as TooltipComponent } from "@recidiviz/design-system";
+import { colors } from "../UiLibrary";
 import { formatAsPct, formatAsNumber } from "../utils";
 
-const TooltipWrapper = styled.div`
+const TooltipWrapper = styled(TooltipComponent)`
```

Comment:
Not entirely sure if I'm understanding the task at hand correctly. Here I based our Tooltip's outermost component from the Tooltip used in our design library. We apply a fair bit of styling changes from the design library's tooltip but this should preserve the look and feel of it to where it was at before.

Reply:
I think part of the goal here was to eliminate any minor look-and-feel differences that don't serve a real purpose, like the slightly different background color for example. (The Tooltip component in the design system is a slightly later iteration of the Spotlight design, so this could be considered an "upgrade")

Styles that actually extend functionality beyond what the DS supports or are tied to the Spotlight-specific contents of tooltips are still fair game.

Reply:
And what should the mobile version of the tooltip look like? Should we keep it the same style as well?

<img width="400" alt="Screen Shot 2022-04-13 at 10 58 14 AM" src="https://user-images.githubusercontent.com/5040739/163242202-e037ee32-36f8-4ffb-8cc6-69805e5a0669.png">
<img width="403" alt="Screen Shot 2022-04-13 at 10 58 36 AM" src="https://user-images.githubusercontent.com/5040739/163242204-818264f4-08e3-499f-a09b-07ac1ed3f4b1.png">

Reply:
ah right. that is something that hasn't been incorporated into the design system yet so you could either preserve it as is or update it to resemble the new design more (which probably would just involve changing the background color?)

Reply:
I am not a designer, but to my eyes the blue-ish color of the tooltip background clashes with the green-ish color scheme of the app, so I'm not sure how to incorporate this new background color with the rest of the app. I would preserve the style as it is for now?

Reply:
yeah I see what you mean ... there's not really any point in doing this task if we are then going to override all of the styles anyway. Which is fine, maybe this is actually a pointless task and we shouldn't do it! Our design system, as we are soon due to discuss, is not in a very good state and this is perhaps an example of that

Reply:
Yeah, I don't really have context regarding what direction we're going with our design system, perhaps we should hold off on this task for now?

Reply:
agreed! sorry for the dead-end task. i would just close the PR and the issue entirely at this point tbh

Reply:
Haha well at least I've started to poke around in this codebase!